app/views/content/life-as-a-teacher/explore-subjects/chemistry/_article.html.erb

```diff
@@ -65,7 +54,7 @@
       <li>chemical and allied industries</li>
       <li>earth and atmospheric science</li>
     </ul>
-    <p><a href="https://www.thenational.academy/teachers/programmes/chemistry-secondary-ks4-foundation-aqa/units">Explore what teaching a key stage 4 chemistry lesson would be like</a>.</p>
+    <p><a href="https://www.thenational.academy/teachers/programmes/chemistry-secondary-ks4-foundation-aqa/units">Explore what teaching a key stage 4 <%= @front_matter["subject"] %> lesson would be like</a>.</p>
```

Comment:
@Sarah-DfE As this will essentially be the template page now, do you want to just tweak this link text to make it clear it is going to the Oak National Academy website?